src/types/ColorFormats.ts

```diff
@@ -4,5 +4,15 @@
  * `hex` returns a color in hexadecimal format. `#000000`
  *
  * `rgbArray` returns a array of rgb. `[0, 0, 0]`
+ *
+ * `hslString` returns an hsl string format. `hsl(0, 0, 0)`
+ *
+ * `hslArray` returns a array of hsl. `[0, 0, 0]`
  */
-export type ColorFormats = 'rgbString' | 'hex' | 'rgbArray';
+export type ColorFormats =
+  | 'rgbString'
+  | 'hex'
+  | 'rgbArray'
+  | 'hslString'
+  | 'hslArray'
+  | 'keyword';
```

Comment:
I think that need to explain the 'keyword' color format.

Reply:
Updated the Readme.